### DISCUSSION TITLE
Automatic mailing-list migration emails

### DISCUSSION THREAD

**Discussion by GregVernon** [snl-dakota/dakota#10] [Ideas]
Some years ago, INL migrated their Google Groups mailing list for their MOOSE code to GitHub Discussions.  They setup a friendly, automatic email directing folks who might email the mailing list to the Discussions page.  Here's an example exchange from as recently as yesterday:

![image](https://user-images.githubusercontent.com/868479/202045889-70a5eeaa-53bf-4d8e-b09a-d253dda4d877.png)

Which did succeed in directing the individual to [post their question](https://github.com/idaholab/moose/discussions/22694) on the discussions page for MOOSE.  A similar mechanism may be beneficial for Dakota's transition to GitHub Discussions, if not already implemented.

**Comment by briadam** [comment]
Thanks Greg, you read our collective Dakota mind. I'm about to (well, soon) configure Dakota's mailman accordingly! Really appreciate the suggestion and evidence of success!

**Reply by briadam** [reply]
I'm also going to add info on how to unsubscribe from dakota-users to the footer of the email.
